sync/class.jetpack-sync-client.php

```diff
@@ -853,5 +920,12 @@ function set_defaults() {
 
 	function reset_data() {
 		$this->reset_sync_queue();
+		// Lets delete all the other fun stuff like transints.
```

Comment:
I think we should have an empty newline above the comment and remove the empty newline below the comment.